tests/driver_dcf77/Makefile

```diff
@@ -0,0 +1,6 @@
+include ../Makefile.tests_common
+
+USEMODULE += dcf77
+BOARD_BLACKLIST := += msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
```

Comment:
Just use `FEATURES_BLACKLIST += arch_msp430` instead.

Reply:
that doesn't work.

Reply:
You might have to `git rebase master`, this was just added recently.